component/block_cache/block_cache.go

```diff
@@ -1998,4 +2000,9 @@ func init() {
 
 	strongConsistency := config.AddBoolFlag("block-cache-strong-consistency", false, "Enable strong data consistency for block cache.")
 	config.BindPFlag(compName+".consistency", strongConsistency)
+
+	// Use BindPFlag for cleanup-on-start to match the configuration parameter name
+	// Add a new CLI flag specific for block_cache
+	cleanupOnStart := config.AddBoolFlag("cleanup-on-start", false, "Clear block cache directory on startup if not empty.")
```

Comment:
We can export this flag from mount.go in generic way and not from file-cache and block-cache. Also remove code related to clean-up-on-start from file-cache.